gym/lib/gym/generators/build_command_generator.rb

```diff
@@ -46,26 +46,25 @@ def options
         options
       end
 
-      def actions
+      def buildactions
```

Comment:
renamed to match more closely with `xcodebuild`'s own [documentation](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/xcodebuild.1.html):

```
buildaction ...
           Specify a build action (or actions) to perform on the target. Available build actions are:
```